liveblog.php

```diff
@@ -1177,6 +1180,32 @@ private static function add_default_plupload_settings() {
 
 			$wp_scripts->add_data( 'wp-plupload', 'data', $script );
 		}
+		
+		/**
+		 * Get TinyMCE editor stylesheet
+		 *
+		 * @return string
+		 */
+		private static function get_tinymce_editor_stylesheet() {
+			$suffix = SCRIPT_DEBUG ? '' : '.min';
+			$version = 'ver=' . get_bloginfo( 'version' );
+			
+			// Default stylesheets from WordPress core
+			$mce_css = includes_url( "css/dashicons$suffix.css?$version" ) . ',' . includes_url( "js/tinymce/skins/wordpress/wp-content.css?$version" );
```

Comment:
use `add_query_arg()` to build the URL.

Reply:
This code is copied from the `_WP_Editors` class. Is there a way we can read the value from that without copy/pasting the code?

If not, please document where this code comes form and what it does.

Reply:
Yes, it's copied from `_WP_Editors ` class, and since it's part of a private function I don't think we can read the value. 

My initial approach is to only use `get_editor_stylesheets` function but that doesn't include default stylesheets that WordPress adds.